tsconfig.json

```diff
@@ -9,13 +9,15 @@
     "noImplicitAny": true,
     "jsx": "react",
```

Comment:
jsx allows `react` or `react-native` 
Should there be a check for special react native build steps?

like `if(buildReactNative) { tsconfig.jsx = 'react-native'; }` ? 

See https://www.typescriptlang.org/docs/handbook/jsx.html